package.json

```diff
@@ -19,10 +20,11 @@
     "cors": "^2.8.5",
     "express": "^4.18.2",
     "jsonwebtoken": "^9.0.2",
-    "mongoose": "^7.5.3",
+    "mongoose": "^6.12.2",
     "nodemailer": "^6.9.5"
   },
   "devDependencies": {
+    "cross-env": "^7.0.3",
```

Comment:
Should this be a devDependency?

Reply:
Yes, its main functionality is to improve development, it has no changes in production